CIF-831 - Make AEM GraphQL "authoring" connector opensource

cjelger opened the pull request.

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- temporarily skip unit test

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- temporarily remove unit test because granite dependency is not publicly available

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- added an "all" package for easy installation of everything

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- fix artifactId of the "all" project

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- install bundles in /apps/bundles/install/

mhaack commented on all/pom.xml:
```diff
@@ -0,0 +1,255 @@
+                    </subPackages>
+                </configuration>
+            </plugin>
+            <plugin>
```
Is this plugin still needed?

cjelger replied:
Yes, this is required to be able to do `content-package:install`

mhaack replied:
Oh right.

mhaack commented on content/cif-connector/pom.xml:
```diff
@@ -0,0 +1,304 @@
+            </plugin>
+
+            <plugin>
+                <groupId>com.day.jcr.vault</groupId>
```
This should use the `filevault-package-maven-plugin` to build the package as well I think.

mhaack commented on content/cif-virtual-catalog/pom.xml:
```diff
@@ -0,0 +1,306 @@
+            
+            <plugin>
+                <groupId>com.day.jcr.vault</groupId>
+                <artifactId>content-package-maven-plugin</artifactId>
```
This should be using the `filevault-package-maven-plugin` as well.

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- fix plugin

cjelger commented:
@mhaack actually I removed the `filevault` plugin and replaced it with the content package plugin ... I had copied that from Mark, I don't know what the difference is between the two, they seem to do exactly the same.

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- fix plugin

cjelger committed:
CIF-831 - Make AEM GraphQL "authoring" connector opensource

- added missing license file
